scripts/generate-report.py

```
@@ -433,32 +954,89 @@ def build_executive_summary(doc, df, epoch, stats, date_label):
-    add_styled_table(doc, ["Parameter", "Value"], weight_rows)
+    weight_rows.append(("Vote Count", format_int(epoch.get("vote_count", "?"))))
+    add_styled_table(
+        doc,
+        ["Parameter", "Value"],
+        weight_rows,
+        col_widths=[Inches(4.7), Inches(2.0)],
+        numeric_cols={1},
+    )
+
+
+def build_methodology_block(doc, epoch, stats, date_label):
```

Ruff (0.15.5)

[warning] Unused function argument: `date_label`

(ARG001)